.gitignore

```diff
@@ -1,6 +1,8 @@
+.DS_Store
```

Comment:
Wait... I thought it was a DS_Store file, Seems like I'm still waking up! Sorry buddy :(.